test/apps/iodemo/ucx_wrapper.h

```diff
@@ -86,6 +86,9 @@ class UcxContext {
 
     void progress();
 
+    static const std::string sockaddr_str(const struct sockaddr* saddr,
```

Comment:
why is this move needed?

Reply:
to make it public and call outside of wrapper from io_demo